tensorflow/python/keras/wrappers/scikit_learn.py

```diff
@@ -330,7 +330,10 @@ def predict(self, x, **kwargs):
             Predictions.
     """
     kwargs = self.filter_sk_params(Sequential.predict, kwargs)
-    return np.squeeze(self.model.predict(x, **kwargs))
+    preds = np.array(self.model.predict(x, **kwargs))
```

Comment:
Recommend use `np.asarray` instead of `np.array` to avoid unnecessary copy.

Reply:
Thank you for the feedback. Just made a new commit as per your suggestion.